providers/implementations/signature/dsa_sig.c

```diff
@@ -343,6 +445,66 @@ static int dsa_verify(void *vpdsactx, const unsigned char *sig, size_t siglen,
     return DSA_verify(0, tbs, tbslen, sig, siglen, pdsactx->dsa);
 }
 
+static int dsa_verify_set_sig(void *vpdsactx,
+                              const unsigned char *sig, size_t siglen)
+{
+    PROV_DSA_CTX *pdsactx = (PROV_DSA_CTX *)vpdsactx;
+    OSSL_PARAM params[2];
+
+    params[0] =
```

Comment:
Hmmm when I said we could fix this later I didnt realise you were already doing forced memcpy that wasnt there before...

Reply:
It can still be fixed later, no?